Not installing via git clone so setuptools_scm issues LookupError. This patch instead sets it manually.

Author: Viktor Rehnberg (Chalmers University of Technology)

diff -ruN Eureka.old/pyproject.toml Eureka/pyproject.toml
--- Eureka.old/pyproject.toml	2023-06-29 16:00:53.700496379 +0200
+++ Eureka/pyproject.toml	2023-06-29 15:56:20.323431241 +0200
@@ -1,6 +1,3 @@
 [build-system]
-requires = ["setuptools>=42", "wheel", "cython", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools>=42", "wheel", "cython"]
 build-backend = "setuptools.build_meta"
-
-[tool.setuptools_scm]
-write_to = "src/eureka/version.py"
diff -ruN Eureka.old/src/eureka/version.py Eureka/src/eureka/version.py
--- Eureka.old/src/eureka/version.py	1970-01-01 01:00:00.000000000 +0100
+++ Eureka/src/eureka/version.py	2023-06-29 15:57:08.240618076 +0200
@@ -0,0 +1 @@
+__version__ = '0.9'
